cfme/tests/configure/test_tag.py

```diff
@@ -24,3 +29,81 @@ def test_tag_crud(category):
     with update(tag):
         tag.display_name = fauxfactory.gen_alphanumeric(32)
     tag.delete(cancel=False)
+
+
+class TestTagsViaREST(object):
+
+    @pytest.fixture(scope="function")
+    def categories(self, request, rest_api, num=3):
+        return _categories(request, rest_api, num)
+
+    @pytest.fixture(scope="function")
+    def tags(self, request, rest_api, categories):
+        return _tags(request, rest_api, categories)
+
+    @pytest.mark.uncollectif(lambda: version.current_version() < '5.5')
+    def test_edit_tags(self, rest_api, tags):
+        if "edit" not in rest_api.collections.tags.action.all:
+            pytest.skip("Edit tags action is not implemented in this version")
+
+        new_names = []
+        tags_data_edited = []
+        for tag in tags:
+            new_name = fauxfactory.gen_alphanumeric().lower()
+            new_names.append(new_name)
+            tag.reload()
+            tags_data_edited.append({
+                "href": tag.href,
+                "name": "test_tag_{}".format(new_name),
+            })
+        rest_api.collections.tags.action.edit(*tags_data_edited)
+        for new_name in new_names:
+            wait_for(
+                lambda: rest_api.collections.tags.find_by(name=new_name),
+                num_sec=180,
+                delay=10,
+            )
+
+    @pytest.mark.uncollectif(lambda: version.current_version() < '5.5')
+    def test_edit_tag(self, rest_api, tags):
+        if "edit" not in rest_api.collections.tags.action.all:
+            pytest.skip("Edit tags action is not implemented in this version")
+
+        tag = rest_api.collections.tags.get(name=tags[0].name)
+        new_name = 'test_tag_{}'.format(fauxfactory.gen_alphanumeric())
+        tag.action.edit(name=new_name)
+        wait_for(
+            lambda: rest_api.collections.tags.find_by(name=new_name),
+            num_sec=180,
+            delay=10,
+        )
+
+    @pytest.mark.meta(blockers=[1290783])
```

Comment:
@lehinevych I think we are now in the time when bugs don't work properly because the release_flag gets set to next version. Use `[BZ(1290783, forced_streams=["5.5"])]` to circumvent this glitch.